lib/elasticsearch/drain/autoscaling.rb

```diff
@@ -41,6 +45,17 @@ def describe_instances
         @instances = instances
       end
 
+      def describe_autoscaling_group
+        group = []
+        groups = @asg_client.describe_auto_scaling_groups(
+          auto_scaling_group_names: [asg]
+        )
+        groups.auto_scaling_groups.each do |page|
```

Comment:
If we're just going to return the first ASG instance, why not just return `groups.auto_scaling_groups.first` instead?